queries.sql

```diff
@@ -111,3 +111,67 @@ LEFT JOIN animals ON owners.id = animals.owner_id
 GROUP BY owners.id
 ORDER BY count DESC
 LIMIT 1;
+
```

Comment:
[ OPTIONAL] 
For better documentation, I suggest you put the query questions as comments at the top of each query block